src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java

```diff
@@ -0,0 +1,108 @@
+package org.folio.marc4ld.mapper.field111;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Map.entry;
+import static org.folio.ld.dictionary.PropertyDictionary.AFFILIATION;
+import static org.folio.ld.dictionary.PropertyDictionary.AUTHORITY_LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.CONTROL_FIELD;
+import static org.folio.ld.dictionary.PropertyDictionary.DATE;
+import static org.folio.ld.dictionary.PropertyDictionary.EQUIVALENT;
+import static org.folio.ld.dictionary.PropertyDictionary.FIELD_LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.LINKAGE;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME;
+import static org.folio.ld.dictionary.PropertyDictionary.PLACE;
+import static org.folio.ld.dictionary.PropertyDictionary.SUBORDINATE_UNIT;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.IDENTIFIER;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_LCCN;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.INSTANCE;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.MEETING;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.WORK;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.mapper.test.MonographTestUtil;
+import org.folio.marc4ld.mapper.test.SpringTestConfig;
+import org.folio.marc4ld.service.ld2marc.Bibframe2MarcMapperImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class BibframeToMarc111IT {
+
+  @Autowired
+  private Bibframe2MarcMapperImpl bibframe2MarcMapper;
+
+  @Test
+  void shouldMapField110() {
+    //given
+    var resource = createResourceWithWorkWith111();
+    var expectedMarc = loadResourceAsString("fields/111/marc_111.jsonl");
+
+    //when
+    var result = bibframe2MarcMapper.toMarcJson(resource);
+
+    //then
+    AssertionsForClassTypes.assertThat(result)
+      .isEqualTo(expectedMarc);
```

Comment:
minor: can be single line statement

Reply:
left as is to align with other similar tests